evalai/utils/auth.py

```diff
@@ -12,9 +12,10 @@
 )
 from evalai.utils.urls import URLS
 
-
 requests.packages.urllib3.disable_warnings()
 
+colors = {'links': 'blue', 'token': 'green', 'error': 'red'}
```

Comment:
What do you think about renaming `token` to something like `information` or `data`? Then it can be used by other commands as well.

Reply:
I think we should use another colour for `information`

But if you say so, I'll do that.

Reply:
Did the changes.